api/envoy/extensions/filters/http/compressor/v3/compressor.proto

```diff
@@ -96,7 +96,8 @@ message Compressor {
       [deprecated = true, (envoy.annotations.deprecated_at_minor_version) = "3.0"];
 
   // Runtime flag that controls whether the filter is enabled or not. If set to false, the
-  // filter will operate as a pass-through filter. If not specified, defaults to enabled.
+  // filter will operate as a pass-through filter, unless overridden by
```

Comment:
Should the comment for [enabled](https://github.com/envoyproxy/envoy/blob/b3bf16a03f02851cd17ef15749c0371f7d3771e3/api/envoy/extensions/filters/http/compressor/v3/compressor.proto#L34) be updated as well?

Reply:
Done.